src/libraries/System.Runtime/tests/System/TimeZoneInfoTests.cs

```diff
@@ -1819,6 +1819,68 @@ public static void IsDaylightSavingTime_CasablancaMultiYearDaylightSavings(strin
             Assert.Equal(offset, s_casablancaTz.GetUtcOffset(dt));
         }
 
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindows))]
```

Comment:
Nit: this could be done with `PlatformSpecificAttribute` using `~TestPlatforms.Windows`. 

https://github.com/dotnet/arcade/blob/0d3414187657bcc31ff4b02f34d9fbd0e85e18dc/src/Microsoft.DotNet.XUnitExtensions/src/TestPlatforms.cs#L11